### PR TITLE
extend caching ttl considerations

### PR DESCRIPTION
**Pull request by alcohol**
Cache-Control is not the only header that can determine how long a
response is considered valid. Expires is old but still widely in use. If
Cache-Control is not present, Expires should be considered next (if
available) to determine a ttl.

I purposely did not use Utils::getMaxAge() since it does not take into
account max-stale and I was not sure if that function should be modified
or not (with concerns to backwards compatibility etc).

Any feedback?

Refs #55

### CODE REVIEW

File: src/CacheStorage.php [hunk 1]
```diff
@@ -277,12 +277,13 @@ private function getTtl(ResponseInterface $response)
                 $ttl += $maxAge;
             }
 
-            // According to RFC5861 stale headers are *in addition* to any
-            // max-age values.
+            // According to RFC5861 stale headers are *in addition* to any max-age values.
```
Comment (mtdowling) [inline src/CacheStorage.php]: I'm an 80 character limit kind of guy. Can you undo this change?

**Comment by alcohol** [comment]
Done. :-)

**Comment by mtdowling** [comment]
Thanks!
